src/Informer/Verbosity.hpp

```diff
@@ -0,0 +1,32 @@
+// Distributed under the MIT License.
```

Comment:
Can you split off this commit into a separate PR, so it can be used in other PRs.  In doing that, can you also make a LoggingGroup, move the functions in Informer.hpp from UtilitiesGroup to LoggingGroup.